contracts/DandelionOrg.sol

```diff
@@ -406,81 +406,81 @@ contract DandelionOrg is BaseTemplate {
         _createTimeLockPermissions(_acl, timeLock, dandelionVoting, dandelionVoting, tokenBalanceOracle);
     }
 
-    // CACHE FNS
-    function _cacheToken(MiniMeToken _token) internal {
-        Cache storage c = cache[msg.sender];
+    // SAVE FNS
+    function _saveToken(MiniMeToken _token) internal {
+        DeployedContracts storage contracts = deployedContracts[msg.sender];
```

Comment:
Nitpicky but I would call this something like `senderDeployedContracts` to make it clear what this is when reading it mid-function.

Reply:
what you are suggesting here is to change the 'contracts' variable name right?

Reply:
Yeah